cg-spec/editors_draft.html

```diff
@@ -1867,6 +1867,39 @@ <h2>Entailment of <a>RDF-star graphs</a></h2>
       </ul>
     </section>
 
+    <section id="merging">
+      <h2>Merging RDF-star graphs</h2>
+
+      <p>[[[RDF11-MT]]] [[RDF11-MT]] defines the <a>merging</a> of two or more <a>RDF graphs</a> as "[taking their] union after forcing any shared blank nodes, which occur in more than one graph, to be distinct in each graph." It is worth precising that, in the case of <a>RDF-star graphs</a>, any <a>blank node</a> in the <a>constituent terms</a> of that graph is concerned by the definition above, <em>not</em> only those in the subject or object position of some <a>asserted triple</a>.</p>
```

Comment:
Is the almost-duplicate `[[[RDF11-MT]]] [[RDF11-MT]]` necessary? It looks a bit odd in the rendering, and more so here.
```suggestion
      <p>[[[RDF11-MT]]] [[RDF11-MT]] defines the <a>merging</a> of two or more <a>RDF graphs</a> as "[taking their] union after forcing any shared blank nodes, which occur in more than one graph, to be distinct in each graph." It is worth noting that, in the case of <a>RDF-star graphs</a>, any <a>blank node</a> in the <a>constituent terms</a> of that graph is concerned by the definition above, <em>not</em> only those in the subject or object position of some <a>asserted triple</a>.</p>
```

Reply:
[[[RDF11-MT]]] should be expanded by ReSpec to (a reference to) the title of the specification, while [[RDF11-MT]] is a reference to the bibliographic entry.

Reply:
Ah, OK. It looked odder in the DIFF than the PREVIEW, probably because of so many up-arrows for the insertions; I should have looked at both.

I think the one word change I suggested is still necessary. (If I've misunderstood what was meant by the non-word "precising", I'm happy to have another word substitute for my "noting".)

Reply:
Oh.. sorry about that. What I meant was "clarifying", which I think is better than "noting" in this context. Would that work for you?

Reply:
"clarifying" is fine.

Reply:
where you say "is concerned", consider using instead "is governed by".